test/lit/passes/gufa-optimizing_merge-blocks_simplify-locals-notee.wast

```diff
@@ -0,0 +1,58 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s --gufa-optimizing  --merge-blocks --simplify-locals-notee -all -S -o - | filecheck %s
+;;
+;; GUFA manages to generate IR that makes merge-blocks leave a stale type, if we
+;; are not careful. simplify-locals-notee then errors if that stale type is
+;; present, so all the passes are needed to hit this bizarre corner case.
```

Comment:
Not sure what this means... Why do all the passes need to hit this corner case wheh simplify-lcoals-notee errors out?

Reply:
It isn't that the passes each need to do that, but that the testcase must include all those passes, for it to hit the error. I improved the text now.